src/storage/src/hummock/sstable/builder.rs

```diff
@@ -390,6 +390,7 @@ impl<W: SstableWriter, F: FilterBuilder> SstableBuilder<W, F> {
         self.block_metas.last_mut().unwrap().total_key_count += 1;
         if !is_new_user_key || value.is_delete() {
             self.block_metas.last_mut().unwrap().stale_key_count += 1;
+            self.last_table_stats.total_stale_key_count += 1;
```

Comment:
I think this +1 is incorrect. Say the builder is used by a compaction task with 2 input SSTs, and the output SSTs drops no keys. Then the `total_stale_key_count` delta reported to meta should be 0.